javatests/arcs/android/integration/policy/scenarios.arcs

```diff
@@ -12,6 +12,31 @@ meta
 
 import './particles.arcs'
 
+/// Scenario: No data is written
+// // TODO(b/183554828): Ingress restricting should allow the fields mentioned
+// //  in the policy to be persisted.
+// @egressType('GeneralEgress')
+// policy NoDataWrittenPolicy {
+//   @allowedRetention(medium: 'Disk', encryption: false)
+//   from Thing access {
+//     a,
+//     b,
+//     c,
+//     d
+//   }
+// }
+
+// Given a recipe that is compliant with a policy
+// @policy('NoDataWrittenPolicy')
+recipe NoDataWritten
+  // And the recipe ingresses a schema Thing {a, b, c, d}
+  handle1: create
+
+  IngressThing
+    input: handle1
```

Comment:
Let us call the name of the handle connection to be ingress?
```suggestion
    ingress: handle1
```

Can you also change the handle connection in the egress particle to be `egress`?

`input` and `output` confuses me. :)